app/models/html_variant.rb

```diff
@@ -75,6 +75,6 @@ def prefix_all_images
   end
 
   def allowed_image_host?(src)
-    src.start_with?("https://res.cloudinary.com/")
+    src.start_with?("https://res.cloudinary.com/") || src.start_with?("https://#{SiteConfig.app_domain}/")
```

Comment:
```suggestion
    src.start_with?("https://res.cloudinary.com/") || src.start_with?(URL.url)
```
Not a huge difference but using our URL helper is a bit cleaner